app/services/transform_notification.rb

```diff
@@ -18,4 +18,8 @@ def publish(data_dir)
   def client
     Aws::SNS::Client.new
   end
+
+  def message(data_dir)
+    JSON.generate(data_dir: data_dir || nil)
```

Comment:
I'm not sure I understand what the `data_dir || nil` would accomplish.  is this casting `false` to `nil`?  Do we need to consider the case of an empty string?

Reply:
It does nothing in Ruby. But it works great in Python.

The lambda will handle it being either null or blank.